### ISSUE TITLE
rustup warnings

### ISSUE THREAD

**Issue by Gankra** [axodotdev/cargo-dist#127]
```
Run rustup update 1.67.1 && rustup default 1.67.1
info: syncing channel updates for '1.67.1-x86_64-unknown-linux-gnu'
info: latest update on 2023-02-09, rust version 1.67.1 (d5a82bbd2 2023-02-07)
info: downloading component 'cargo'
info: downloading component 'rust-std'
info: downloading component 'rustc'
info: installing component 'cargo'
info: installing component 'rust-std'
info: installing component 'rustc'

info: checking for self-updates
  1.67.1-x86_64-unknown-linux-gnu installed - rustc 1.67.1 (d5a82bbd2 2023-02-07)

warning: tool `rustfmt` is already installed, remove it from `/home/runner/.cargo/bin`, then run `rustup update` to have rustup manage this tool.
warning: tool `cargo-fmt` is already installed, remove it from `/home/runner/.cargo/bin`, then run `rustup update` to have rustup manage this tool.
info: using existing install for '1.67.1-x86_64-unknown-linux-gnu'
info: default toolchain set to '1.67.1-x86_64-unknown-linux-gnu'

  1.67.1-x86_64-unknown-linux-gnu unchanged - rustc 1.67.1 (d5a82bbd2 2023-02-07)
```

Concerning!

**Comment by Gankra** [comment]
Since it's seemingly just rustfmt descyncing I'm unblocking 0.0.3 on this.

**Comment by Gankra** [comment]
This is no longer relevant with the deprecation of rust-toolchain-version in #275
